app/services/course/assessment/answer/multiple_response_auto_grading_service.rb

```diff
@@ -1,7 +1,8 @@
 class Course::Assessment::Answer::MultipleResponseAutoGradingService < \
   Course::Assessment::Answer::AutoGradingService
   def grade(auto_grading)
-    auto_grading.answer.grade, messages = grade_answer(auto_grading.answer.actable)
+    auto_grading.answer.correct, auto_grading.answer.grade, messages =
+      grade_answer(auto_grading.answer.actable)
```

Comment:
Wait, there is no existing documentation for return actually. Do we want to add that ?